Protocol.md

```diff
@@ -162,6 +162,51 @@ responders, the server MUST dynamically assign identifiers (in the range
 of `0x02..0xff`). A server-assigned address becomes invalid as soon as
 the connection to the server has been severed.
 
+# Exchanging Signalling Information
+
+Please note that this section is informational only.
+
+In order to establish a signalling channel using SaltyRTC, the following
+information has to be available to both peers:
+
+* WebSocket URI scheme (`ws` or `wss`),
+* Server's permanent public key (optional but recommended),
+* Server host (as defined in [RFC3986, 3.2.2](https://tools.ietf.org/html/rfc3986#section-3.2.2))
+* Server port (as defined in [RFC3986, 3.2.3](https://tools.ietf.org/html/rfc3986#section-3.2.3))
+* Signalling path, and
+* Authentication token (only if responder and if not trusted)
+
+How this information should be exchanged is deliberately not defined by
+this document. However, we will provide an idea of how the data can be
+encoded and exchanged by extending the WebSocket URI in the following
+way:
+
+```
+<scheme>://<server-host>:<server-port>/<signalling-path>
+?<server-permanent-public-key>#<authentication-token-hex>
+```
```

Comment:
I'd format this by indentation, it's easier to read in ASCII form and the result is the same.